packages/frontend/src/creation-form/index.tsx

```diff
@@ -162,20 +133,18 @@ export const Component = ({
                 // implemented elsewhere, as this component should be completely
                 // spec-agnostic
                 if (
-                    !metric ||
-                    !timestamp ||
-                    !constraintType ||
-                    constraintValues[0] === undefined ||
-                    constraintValues[1] === undefined ||
-                    !payload?.protocol
+                    !state.timestamp ||
+                    !state.specification ||
+                    !state.specification.metric ||
+                    !state.specification.payload ||
+                    !state.constraint ||
+                    !state.constraint.type ||
+                    !state.constraint.value0 === undefined ||
```

Comment:
Wrong negation `!` here and the line below.

Reply:
Whoops, fixed in 5c1230c72d85f95dba57fff2c5d144ac49760776